src/components/BlogArticlePreview/BlogArticlePreview.js

```
@@ -1,37 +1,89 @@
-import React from 'react';
+import React, { Component } from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
 
+import HttpService from '../../service/HttpService/httpService';
+
 import './BlogArticlePreview.scss';
 
-export const CN = 'article-preview';
+export const CN = 'blog';
+
+class BlogArticlePreview extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      blogsList: []
+    };
+  }
+
+  componentDidMount() {
+    this.getBlogs();
+  }
+
+  async getBlogs() {
+    const userAPI = new HttpService();
 
-const BlogArticlePreview = (props) => {
-  const { imageSrc, heading, text } = props;
-  const backgroundImageURL = {
-    backgroundImage: `url(${imageSrc})`
-  };
+    try {
+      const response = await userAPI.get(`${process.env.BASE_URL}/blogs`);
+      if (response && response.data) {
+        this.setState({ blogsList: response.data });
+      }
+    } catch (error) {
+      throw (new Error());
```

why you throw an empty error here instead of logging it to the console?

---

this is not very good practice!